### DISCUSSION TITLE
Stupid little rainbow command for builders

### DISCUSSION THREAD

**Discussion by pruuk** [evennia/evennia#2505] [Community Contribs & Snippets]
I got tired of trying to figure out text colors and how they look on different terminals and mud clients, so I made a builder command to just display the text colors:

class CmdShowColors(Command):
    """
    Sends a text colo palate for every possible numbered color palate to the
    caller, so they can see what colors are mapped to what.

    usage:
        rainbow

    """
    key = "rainbow"
    locks = "cmd: perm(Builders)"
    help_category = "Building"

    def func(self):
        """
        Give me that sweet unicorn puke!
        """
        colors = f" Text Colors by number: "
        for first_digit in range(6):
            for second_digit in range(6):
                for third_digit in range(6):
                    colors += f"|{first_digit}{second_digit}{third_digit}{first_digit}{second_digit}{third_digit}|n "
        self.caller.msg(f"{colors}")

**Comment by pruuk** [comment]
![Screenshot 2021-08-18 195813](https://user-images.githubusercontent.com/72707866/129995506-82e594dd-1c99-486c-8368-fa616d9b81f6.png)

**Comment by dmreichard** [comment]
Doesn't the command `color xterm256` accomplish the same thing?

**Comment by Griatch** [comment]
Yes, check out the `color` command. :wink:
